docs/customizations/footer.md

```diff
@@ -0,0 +1,218 @@
+# Footer
+
+## Table of contents
+
+- [Interfaces](#interfaces)
+  - [IFooterProps](#ifooterprops)
+  - [IFooterComponentOverrides](#ifootercomponentoverrides)
+  - [IFooterControlProps](#ifootercontrolprops)
+  - [IFooterStyleProps](#ifooterstyleprops)
+- [Sample Scenarios](#sample-scenarios)
+  - [Replacing default sub-components with custom components](#replacing-default-sub-components-with-custom-components)
+  - [Using a custom send box in the footer](#using-a-custom-send-box-in-the-footer)
+  - [Overriding default button behaviors](#overriding-default-button-behaviors)
+
+## Interfaces
+
+### [IFooterProps](https://github.com/microsoft/omnichannel-chat-widget/blob/main/chat-components/src/components/footer/interfaces/IFooterProps.ts)
+
+The top-level interface for customizing `Footer`.
+
+| Attribute | Type | Required | Description | Default |
+| - | - | - | - | - |
+| componentOverrides     | [IFooterComponentOverrides](#ifootercomponentoverrides)     | No | Used for overriding default `Footer` components, including all the default buttons | -
+controlProps | [IFooterControlProps](#ifootercontrolprops) | No | Properties that control the element behariors | -
+styleProps | [IFooterStyleProps](#ifooterstyleprops) | No | Properties that control the element styles | -
+
+### [IFooterComponentOverrides](https://github.com/microsoft/omnichannel-chat-widget/blob/main/chat-components/src/components/footer/interfaces/IFooterComponentOverrides.ts)
```

Comment:
missing default https://github.com/microsoft/omnichannel-chat-widget/blob/main/chat-components/src/components/footer/common/defaultProps/defaultFooterComponentOverrides.tsx

Reply:
This is used in storybook only